spring-cloud-netflix-turbine/src/main/java/org/springframework/cloud/netflix/turbine/ConfigurationBasedTurbineClustersProvider.java

```diff
@@ -0,0 +1,28 @@
+package org.springframework.cloud.netflix.turbine;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConfigurationBasedTurbineClustersProvider implements TurbineClustersProvider {
+
+    private final TurbineAggregatorProperties properties;
```

Comment:
moved this configuration from archaius based DynamicProperty to @ConfigurationProperties class